cmd/krew/cmd/update.go

```diff
@@ -124,31 +141,27 @@ func ensureIndexesUpdated(_ *cobra.Command, _ []string) error {
 				returnErr = err
 			}
 		}
-	}
-
-	fmt.Fprintln(os.Stderr, "Updated the local copy of plugin index.")
 
-	if len(preUpdateIndex) == 0 {
-		return nil
+		if os.Getenv(constants.EnableMultiIndexSwitch) == "" || isDefaultIndex(idx.Name) {
+			fmt.Fprintln(os.Stderr, "Updated the local copy of plugin index.")
+		} else {
+			fmt.Fprintf(os.Stderr, "Updated the local copy of plugin index %q.\n", idx.Name)
+		}
```

Comment:
nit: this message is shown even if the `EnusreUpdated()` call failed with an error.

Reply:
Good catch!